app/views/shared/_user.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= stylesheet_link_tag('index') %>
+<div class="container">
+  <div class="user">
+    <img src= <% user.photo %> alt="avatar.jpg"/>
```

Comment:
- [ ] _Well done for using partials 👏👏_ However, despite of using the correct URL for photo, the image is not being displayed. Kindly make sure to use `<%= %>` tags to embed. Kindly fix this. Also please make sure to follow the [given wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_root_users.png?token=GHSAT0AAAAAAB5V373VYYE6N7XFP65YVMMUZIWC27Q) for styling

This is what I have in the terminal 👇
![image](https://github.com/mosidrum/blog-app/assets/106119105/e36f9247-3099-45aa-a53d-ad8395a2025b)

That's how it's currently displayed 👇
![image](https://github.com/mosidrum/blog-app/assets/106119105/eb507af5-b849-426f-8f1e-157515555511)